docs/source/concepts/ledger-model/ledger-structure.rst

```diff
@@ -129,12 +133,12 @@ Thus, when accepting the offer, `A` has to additionally show a valid quality cer
 In the paint offer example, the underlying type of contracts consists
 of three sorts of contracts:
 
-PaintOffer houseOwner painter obligor
-  Intuitively an offer by
+PaintOffer houseOwner painter obligor refNo
+  Intuitively an offer with a reference number by
```

Comment:
In brackets just as it's not the most important detail at this point
```suggestion
  Intuitively an offer (with a reference number) by
```

Reply:
Done.